### ISSUE TITLE
javascript syntax highlighting breaks when using template literals

### ISSUE THREAD

**Issue by benvds** [Microsoft/vscode#17931]
- VSCode Version: 1.8.1
- OS Version: macOS 10.12.2

Steps to Reproduce:

1. Open new editor, set syntax language to javascript
2. Code example that breaks:

```javascript
const apiKey = 'foo';
const keyName = `filters[${apiKey}][]`;

// it's all fine
let subject = { [keyName]: checked };

// syntax highlighting ok
let breakMe = { [`filters[${apiKey}][]`]: 'bar' };

// template literal without let breaks syntax highlighting
breakMe = { [`filters[${apiKey}][]`]: 'bar' };
//                                ^- syntax highlighting breaks here

console.log('syntax broken here as well');
```

![example](http://i.imgur.com/Pzzrhnl.png)

**Comment by rebornix** [comment]
Dup of https://github.com/Microsoft/vscode/issues/13166

**Comment by mjbvz** [comment]
This seems to be fixed in VSCode 1.9 insiders. @benvds, please give [1.9 insiders a try](https://code.visualstudio.com/insiders) and let me know if you still see this issue. 

Thanks

**Comment by benvds** [comment]
@mjbvz seems to work now, thx!

![example](http://i.imgur.com/98GQimQ.png)
